app/controllers/concerns/api/v1/error_handler.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+module Api
+  module V1
+    module ErrorHandler
+      extend ActiveSupport::Concern
+
+      private
+
+      # Render error response in json format
```

Comment:
should it be `Render error response in JSON:API format` to be more specific?

Reply:
Fixed in https://github.com/gutakk/ruby-google-scraper/pull/17